### PR TITLE
Move OpenCV to opt for windows

### PR DESCRIPTION
**Pull request by rasapala**
### 🛠 Summary

JIRA/Issue if applicable.
Describe the changes.

### 🧪 Checklist

- [ ] Unit tests added.
- [x] The documentation updated.
- [ ] Change follows security best practices.
``

### CODE REVIEW

**Review by dkalinowski — APPROVED** [review]

File: docs/windows_developer_guide.md [hunk 1]
```diff
@@ -61,14 +61,6 @@ python --version
 pip install numpy==1.23
 ```
 
-## MSYS
-Download and install in c:\opt\msys64 https://github.com/msys2/msys2-installer/releases/download/2024-07-27/msys2-x86_64-20240727.exe
```
Comment (mzegla) [inline docs/windows_developer_guide.md:65]: Not sure if this part should be permanently deleted.

File: docs/windows_developer_guide.md [hunk 2]
```diff
@@ -61,14 +61,6 @@ python --version
 pip install numpy==1.23
 ```
 
-## MSYS
```
Comment (atobiszei) [inline docs/windows_developer_guide.md:64]: Please also move NPM after proxy setting as this is also required for npm

Comment (atobiszei) [reply docs/windows_developer_guide.md:64]: Actually setting proxy via export/setting variables didn't work. We need first to set it with npm:
nvm proxy http://SOME_PROXY
then use other npm commands

File: docs/windows_developer_guide.md [hunk 3]
```diff
@@ -96,13 +98,6 @@ npm i --global yarn
...
 
-## Building with proxy
-Please set the proxy setting for windows for in environment variables when building behind proxy
-```
-set HTTP_PROXY=my.proxy.com:123
-set HTTPS_PROXY=my.proxy.com:122
-```
-
 ## GET CODE
 ```
```
Comment (atobiszei) [inline docs/windows_developer_guide.md:102]: Add information thath Win11 SDK is required - on screenshot it is unselected but the opencv compilation fails

File: docs/windows_developer_guide.md (hunk 1 above)
Comment (rasapala) [reply docs/windows_developer_guide.md:65]: Changed order in install_deps so that msys is available at the start.

**Review by atobiszei — APPROVED** [review]

**Review by mzegla — APPROVED** [review]

**Pull request merged**